test/browser_tests/spec_suites/large_screen/large_contact-us.js

```diff
@@ -1,24 +0,0 @@
-'use strict';
```

Comment:
If we're getting rid of the screen size spec directories and including them in each page/section spec, we should just drop the shared directory, because it's no longer an accurate description of the directory.